cli/args/package_json.rs

```diff
@@ -131,7 +140,24 @@ impl NpmInstallDepsProvider {
                 });
               }
             }
-            PackageJsonDepValue::Workspace(version_req) => {
+            PackageJsonDepValue::Workspace(workspace_version_req) => {
+              let version_req = match workspace_version_req {
+                PackageJsonDepWorkspaceReq::VersionReq(version_req) => {
+                  version_req
+                }
+                PackageJsonDepWorkspaceReq::Tilde => {
+                  VersionReq::from_raw_text_and_inner(
+                    "workspace:~".to_string(),
+                    RangeSetOrTag::Tag("workspace".to_string()),
```

Comment:
I think these should go to a version requirement that's `*`?

Reply:
Are you saying that this should be "unbounded version range"?

Reply:
Yeah, I believe they match any workspace package with the same name.